themes/houston.json

```
@@ -1960,9 +1960,9 @@
     "pickerGroup.foreground": "#eef0f9",
     "progressBar.background": "#4bf3c8",
     "scrollbar.shadow": "#000000",
-    "scrollbarSlider.activeBackground": "#54b9ff",
-    "scrollbarSlider.background": "#343841",
-    "scrollbarSlider.hoverBackground": "#545864",
+    "scrollbarSlider.activeBackground": "#54b9ff99",
+    "scrollbarSlider.background": "#343841DD",
+    "scrollbarSlider.hoverBackground": "#54586499",
```

What do you think of these tweaks?

```suggestion
    "scrollbarSlider.activeBackground": "#54b9ff66",
    "scrollbarSlider.background": "#54586466",
    "scrollbarSlider.hoverBackground": "#545864B3",
```

Looks like VS Code’s [default opacities](https://github.com/microsoft/vscode/blob/a9c390b23628adedadb0116de2b258524abcb838/src/vs/platform/theme/common/colorRegistry.ts#L278-L280) here are 40% (`66` in hexadecimal) for the background shades, and 70% (`B3` in hex) for the hover shade.

By reducing the opacity but brightening the colours we can let more of the background minimap details through while still keeping a similar contrast between the scrollbar elements.

---

Change makes sense, I will have a look at it when I get home, then commit it! Thank you for looking at it :)